config/locales/en.yml

```diff
@@ -109,7 +109,8 @@ en:
     trainees:
       edit:
         defer: Defer
-        withdraw: Withdraw
+        withdraw: withdraw
```

Comment:
Would this be better done with CSS? The case of these is dependent on the order they are displayed. Probably fine, maybe check with @edwardhorsford

Reply:
I could see this being done with some kind of rails helper as an alternate solution. But medium term I think withdraw will *always* be the second link so should always be lowercase.

Reply:
@gpeng, not sure how CSS would make things easier here, happy to discuss

Reply:
I changed my mind on this - I didn't realise that in _all_ current situations, 'withdraw' is the second option. It's a little strange seeing a mixture of lowercase and uppercase in the locales depending on where the word is in the sentence... but if we refactor into a helper now, we're solving a problem that doesn't really exist.

If the capitalisation becomes dynamic in the future due to an update in design, let's revisit it at the point.